pkg/handler3/handler.go

```diff
@@ -213,10 +215,21 @@ func (o *OpenAPIService) HandleGroupVersion(w http.ResponseWriter, r *http.Reque
 			if clause.SubType != accepts.SubType && clause.SubType != "*" {
 				continue
 			}
+			if accepts.SubType == subTypeProtobufDeprecated {
```

Comment:
Same comment as above, this condition/error can be removed.

Reply:
Done

Reply:
removed printing deprecated message